toqito/matrix_props/is_etf.py

```diff
@@ -0,0 +1,69 @@
+"""Checks if matrix forms an equilangular tight frame (ETF)."""
+
+import numpy as np
+
+from toqito.matrix_ops import vectors_to_gram_matrix
+
+
+def is_etf(mat: np.ndarray, rtol: float = 1e-9, atol: float = 1e-9) -> bool:
+    r"""Check if a matrix constitutes an equilangular tight frame (ETF).
+
+    Definition taken from:
+    :cite:`hoffman2011equilangular`.
```

Comment:
[nit]: Does not need to be on two lines. 
```suggestion
    Definition taken from: :cite:`hoffman2011equilangular`.
```